sdk/resourcemanager/Azure.ResourceManager.Core/tests/Unit/ArmClientOptionsTests.cs

```diff
@@ -67,20 +80,22 @@ public void VersionNonLoadedChanges()
             var apiVersions = "2019-10-01";
             AzureResourceManagerClientOptions options = new AzureResourceManagerClientOptions();
             options.ApiVersions.SetApiVersion("Microsoft.Logic/LogicApps", apiVersions);
-            var result = options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps");
+            string result;
+            options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps", out result);
             Assert.True(result.Equals(apiVersions));
 
             apiVersions = "2021-02-01";
             options.ApiVersions.SetApiVersion("Microsoft.Logic/LogicApps", apiVersions);
-            result = options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps");
+            options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps", out result);
             Assert.True(result.Equals(apiVersions));
         }
 
         [TestCase]
         public void TestKeyDoesNotExist()
         {
             AzureResourceManagerClientOptions options = new AzureResourceManagerClientOptions();
-            var result = options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps");
+            string result;
+            options.ApiVersions.TryGetApiVersion("Microsoft.Logic/LogicApps", out result);
```

Comment:
I think from the perspective of the user this method means "Tell me what version my current client is going to use for Microsoft.Logic/LogicApps"

If that is the case our answer should not be null.  Do you think this method means something different?

Reply:
this is the tryGet to check if the variable is set with no API call. So LogicApps is not a loaded extensions, and was never set by the user, hence it would be null.